docker/ci/build

```diff
@@ -2,6 +2,7 @@
 
 set -eux
 
+PATH=$PATH:/usr/local/bin
```

Comment:
Suggest double quotes here so spaces in the PATH couldn't be an issue.

Reply:
Added quotes!